tbtc-ts/test/sweep.test.ts

```diff
@@ -0,0 +1,460 @@
+import TBTC from "./../src"
+import { BigNumber } from "ethers"
+import { RawTransaction, UnspentTransactionOutput } from "../src/bitcoin"
+import {
+  sweepWithPreviousSweep,
+  sweepWithNoPreviousSweep,
+  testnetDepositScripthashAddress,
+  testnetDepositWitnessScripthashAddress,
+  testnetWalletAddress,
+  testnetWalletPrivateKey,
+} from "./data/bitcoin"
+import { MockBitcoinClient } from "./utils/mock-bitcoin-client"
+// @ts-ignore
+import bcoin from "bcoin"
+import * as chai from "chai"
+import chaiAsPromised from "chai-as-promised"
+chai.use(chaiAsPromised)
+import { expect } from "chai"
+
+describe("Sweep", () => {
```

Comment:
If we decide to switch to `mainUtxo` naming, let's make sure we change everything here accordingly.

Reply:
Done in Done in https://github.com/keep-network/tbtc-v2/pull/96/commits/4dccc33c9feb9252e2577f323da7294a691b3569.